client/dead.lua

```diff
@@ -185,9 +190,15 @@ CreateThread(function()
                         DrawTxt(0.90, 1.40, 1.0, 1.0, 0.6, Lang:t('info.help_requested'), 255, 255, 255, 255)
                     end
 
+                    if not IsEmsOnDuty() then
```

Comment:
Doesn't this get spammed since this is inside of a loop? Don't think players want a screen filled with notify

Reply:
This is fair, my workaround in the next commit will be to implement a check directly into the respawn timer, enabling you to just press the respawn button provided that there is no EMS on duty.